packages/notebook/src/browser/view-model/notebook-model.ts

```diff
@@ -71,6 +71,15 @@ export class NotebookModel implements Saveable, Disposable {
     @inject(UndoRedoService)
     private readonly undoRedoService: UndoRedoService;
 
+    @inject(NotebookModelProps)
+    private props: NotebookModelProps;
+
+    @inject(MonacoTextModelService)
+    modelService: MonacoTextModelService;
+
+    @inject(NotebookCellModelFactory)
+    private cellModelFactory: (props: NotebookCellModelProps) => NotebookCellModel;
```

Comment:
Suggestion: It might make sense to introduce a dedicated type for that.
```ts
type NotebookCellModelFactory = (props: NotebookCellModelProps) => NotebookCellModel;
...
private cellModelFactory: NotebookCellModelFactory;
```